README.md

```diff
@@ -1 +1,360 @@
 # Centrapay Documentation
+
+## Overview
+
+### Introduction
+
+Welcome to Centrapay! Payments through Centrapay are accomplished via a two-step process, involving both the merchant and consumer.  
+
+The API is designed to support payment terminals, point of sale, shopping cart, vending and unattended transactions.
+
+It enables consumers to pay with a range of digital assets, including bitcoin and vouchers. The merchant can initiate payment from a consumer or the consumer can push a payment to the merchant. It will also be possible to activate our services in the payment flow such as carbon offsetting.
+
+### Payment Flow
+
+#### Payment flow using Pocket Vouchers
+
+```
+requests.create > Pocket Voucher Code Entry screen on terminal > User enters Pocket Voucher Code into terminal > requests.pay > terminal Loading >Webhook Notification > terminal display result.
+```
+
+#### Payment flow using QR Code
+
+```
+requests.create > Generate QR Code on terminal > User selects payment type on phone > Phone Scans > requests.pay > Authenticated and Processed via Phone > Webhook Notification > terminal display result.
+You will need to create a request ‘/requests.create’ first and use the requestId for making a payment ‘/requests.pay’.
+```
+
+### Glossary
+
+### API keys and access
+
+To gain access to the payments API, please create an account in our Portal. Once you’ve completed the sign up process and acknowledged our terms, we will provide a merchant_id, secret, and api_key via the Portal.
+
+To generate test credentials, click here.
+
+
+### API protocols
+
+In order to be authorized to use any of the payments API endpoints, you will need to send your api_key in the header of your request.
+
+
+### API hosts
+
+service.centrapay.com/payments/api
+
+### Endpoints
+
+|             |                                                                                                   |
+|-------------|---------------------------------------------------------------------------------------------------|
+| Requests    | /requests.cancel <br> /requests.create <br> /requests.info <br> /requests.pay <br> /requests.void |
+| Service     | /service.info                                                                                     |
+| Transaction | /transactions.processSettlements <br> /transactions.refund                                        |
```

Comment:
We should probably remove .processSettlements as I have no idea when the last time anyone used that and the shape it's in... even not 100% on what it does currently.